Jenkinsfile

```diff
@@ -417,21 +415,26 @@ def inAndroidContainer(workerFunction) {
 }
 
 def buildAndroid() {
-  inAndroidContainer {
-    sh 'npm ci --ignore-scripts'
-    sh 'cd react-native/android && ./gradlew publishAndroid'
-    sh 'npm pack'
-    stash includes: 'realm-*.tgz', name: 'android'
+  return {
+    myNode('docker') {
+      unstash 'source'
+      def image = buildDockerEnv('ci/realm-js:android-build', '-f Dockerfile.android')
+      image.inside('-e HOME=/tmp') {
+        sh "./scripts/nvm-wrapper.sh ${nodeTestVersion} npm ci --ignore-scripts" //using --ignore-scripts to skip building for node
+        sh "./scripts/nvm-wrapper.sh ${nodeTestVersion} node scripts/build-android.js"
+        sh "./scripts/nvm-wrapper.sh ${nodeTestVersion} npm pack ."
```

Comment:
this will create only one block in Jenkins and combine the shell commands. We are doing it elsewhere in the Jenkins script as well and I suspect it will not improve much on the performance.